boot.php

```diff
@@ -1,5 +1,22 @@
 <?php
 
+namespace FriendsOfRedaxo\neues;
+
```

Comment:
Richter wäre wohl `namespace FriendsOfRedaxo\Neues;`

Reply:
Du kannst direkt Code-Vorschläge über einen Klick auf diese Schaltfläche hinzufügen.

![image](https://github.com/FriendsOfREDAXO/neues/assets/3855487/2724dbfa-bfa4-45a7-ad03-92e9d8321caa)

Das sieht dann so aus:

```suggestion
namespace FriendsOfRedaxo\Neues;
```